### DISCUSSION TITLE
How do I remove a payment method from github?

### DISCUSSION THREAD

**Discussion by nullptr818** [community/community#65492] [New to GitHub]
Hey there. I'm trying to delete the payment method I added is there a way to delete?

**Comment by unknown4now** [comment]
> Hey there. I'm trying to delete the payment method I added is there a way to delete?

There use to be a remove button but I guess it's not possible anymore. Yeeah I would like to remove that as well. I just tried and not letting me delete...

**Comment by Gihansachith92** [comment]
A delete button once existed, but I believe it is no longer functional. Yeah, I want to take that out too. I tried to erase something, but it won't let me.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
